src/bsoncxx/builder/core.hpp

```diff
@@ -263,6 +268,11 @@ class BSONCXX_API core {
     void append(const oid& value);
 
     ///
+    /// Append a decimal128 object as a BSON Decimal128.
+    ///
+    void append(const decimal128& value);
```

Comment:
Right!  I thought you were talking about the b_ version.  I'll amend the plain decimal128 one.  (This is an example of the problem too many types with similar names!)